app/services/force/ami_service.rb

```diff
@@ -0,0 +1,13 @@
+module Force
+  # encapsulate all Salesforce AMI querying functions
+  class AmiService < Force::Base
+    def ami(params)
+      response = api_get('/ami', params)
+      amis = []
+      response.each do |ami|
+        amis.push(chartType: ami[:chartType], year: ami[:year], householdSize: ami[:numOfHousehold], amount: ami[:amount])
```

Comment:
suggestion: format this to multi line like
```
{ 
 chartType: ami[:chartType]
....
}
```
OR
can you return `numOfHousehold` as the key instead of `householdSize` ? and then you can just use `ami.slice(:chartType, :year, :numOfHousehold,...)`